modules/UI/UI.js

```diff
@@ -227,8 +231,8 @@ UI.initConference = function () {
 
     APP.store.dispatch(checkAutoEnableDesktopSharing());
 
-    if(!interfaceConfig.filmStripOnly) {
-        Feedback.init(eventEmitter);
+    if (interfaceConfig.filmStripOnly) {
+        APP.store.dispatch(shouldShowPostCallFeedbackDialog(false));
```

Comment:
This logic isn't about initializing the feedback button, though. It's about automatically showing (or not showing) the feedback dialog at the end of the call, which is independent of the feedback button. If I've misunderstood, could you try explaining your feedback again?